src/main/java/beam/analysis/plots/GraphSurgePricing.java

```diff
@@ -445,8 +446,8 @@ private void writePriceSurgeCsv(double[][] dataset, List<String> categoriesList,
     }
 
     private void writeTazCsv(Map<String, double[][]> dataset) {
-
-        try (BufferedWriter out = new BufferedWriter(new FileWriter(new File(surgePricingAndRevenueWithTaz)))) {
+        // try (BufferedWriter out = new BufferedWriter(new FileWriter(new File(surgePricingAndRevenueWithTaz)))) {
```

Comment:
No need to keep commented code around - let history handle any questions.

Reply:
agree